### PR TITLE
Update database value for softmasked sequences

### PR DESCRIPTION
**Pull request by jyothishnt**
## Description
Changes to blast config to use softmaksed sequences

## Related JIRA Issue(s)
https://www.ebi.ac.uk/panda/jira/browse/ENSWBSITES-1784

## Deployment URL(s)
http://new-blast-db.review.ensembl.org

## Views affected
BLAST

### CODE REVIEW

File: tests/fixtures/blast/blastSettingsConfig.json [hunk 1]
```diff
@@ -590,16 +594,27 @@
...
   },
   "database_sequence_types": {
+    "dna_sm": "dna",
     "dna": "dna",
     "cdna": "dna",
     "pep": "protein"
   },
   "programs_configurator": [
+    {
+      "sequence_type": "dna_sm",
```
Comment (azangru) [inline tests/fixtures/blast/blastSettingsConfig.json:604]: This (and same in another entry below) is probably wrong. Look at the configurator for protein below — it says `sequence_type: "protein"`, rather that `sequence_type: "pep"`.

I actually suspect that these two entries that you added to the configurator are unnecessary. Need to check and update Andres's config.

Comment (jyothishnt) [reply tests/fixtures/blast/blastSettingsConfig.json:604]: Doh! right, this is sequence type and not database type!

**Review by azangru — APPROVED** [review]
Generally looks good.

Due to this change, however, we can't deploy the client before the server. We'll need to tell Andres to deploy the updated tools api on staging first – and merge this PR right after that. Same with deploying to production.

Screenshot 2022-12-08 at 14 09 50

**Comment by jyothishnt** [comment]
Tested on my sandbox and it works fine with the current blastconfig in staging. Means we can merge this without breaking. @azangru

**Review by imransl — APPROVED** [review]

**Pull request merged**